zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/ProcessDefinitionQueryControllerTest.java

```diff
@@ -206,4 +223,75 @@ public void shouldGetProcessDefinitionXmlHasNoXml() {
     // Verify that the service was called with the valid key
     verify(processDefinitionServices).getProcessDefinitionXml(23L);
   }
+
+  @Test
+  public void shouldReturnFormItemForValidFormKey() throws Exception {
+    when(processDefinitionServices.getByKey(1L))
+        .thenReturn(
+            new ProcessDefinitionEntity(
+                1L, "name", "id", "xml", "resource", 1, "tag", "tenant", "formId"));
+    when(formServices.getLatestVersionByFormId("formId"))
+        .thenReturn(Optional.of(new FormEntity("0", "tenant-1", "formId", "schema", 1L)));
+
+    webClient
+        .get()
+        .uri(PROCESS_DEFINITION_URL + "1/form")
+        .accept(MediaType.APPLICATION_JSON)
+        .exchange()
+        .expectStatus()
+        .isOk()
+        .expectBody()
+        .json(FORM_ITEM_JSON);
+
+    verify(processDefinitionServices, times(1)).getByKey(1L);
+    verify(formServices, times(1)).getLatestVersionByFormId("formId");
+  }
+
+  @Test
+  public void shouldReturn404ForFormInvaliProcessKey() throws Exception {
+    when(processDefinitionServices.getByKey(999L))
+        .thenThrow(new NotFoundException("Process Definition with key 999 not found"));
```

Comment:
This is a nitpick but I would lowercase definition, so "Process definition".

Reply:
Hi @christinaausley, Thanks for the suggestion, this is a test class and this behaviour is only threw on tests. I was checking the another test classes, and they are following this standard, for example Process Instance on `ProcessInstanceQueryControllerTest`

Same standard on all services, So this would require a change in general. @ralfpuchert  which standard should we follow over the error messages? 

I will keep as is for now and we can change all together in a different issue if it is the case @christinaausley

Reply:
I prefer lowercase definition, in this case "Process definition". We can change this step by step in all beginning here.